### DISCUSSION TITLE
Improvement - "Advanced claim to group" idp mapper

### DISCUSSION THREAD

**Discussion by artur-baltabayev** [keycloak/keycloak#8538] [Ideas]
Hello Keycloak community,

we implemented a new "Advanced claim to group" idp mapper, which functions similarly to the already existing
"Advanced claim to role" mapper. Instead of assigning roles to users, the new mapper assigns users to groups (if the selected claims are present). We chose not to implement a base version of the new mapper, because it only would introduce redundant functionality.

Link to the PR : https://github.com/keycloak/keycloak/pull/8467

Any feedback is greatly appreciated.

**Comment by annasarte** [comment]
Hello @artur-baltabayev . Can you please look at my question? https://github.com/keycloak/keycloak/discussions/13646  I am having trouble using this feature that you included (I assume your PR above is what makes me see this feature in Keycloak 18.0.2.).  Thanks in advance.
